### ISSUE TITLE
rails generate react:install not working with Webpacker

### ISSUE THREAD

**Issue by brentdodell** [reactjs/react-rails#775]
Help us help you! Please choose one:

- [ ] My app crashes with `react-rails`, so I've included the stack trace and the exact steps which make it crash.
- [ ] My app doesn't crash, but I'm getting unexpected behavior. So, I've described the unexpected behavior and suggested a new behavior.
- [x] I'm trying to use `react-rails` with another library, but I'm having trouble. I've described my JavaScript management setup (eg, Sprockets, Webpack...), how I'm trying to use this other library, and why it's not working.
- [ ] I have another issue to discuss.

-------

`rails generate react:install` is broken, I believe by [this](https://github.com/rails/webpacker/pull/636) Webpacker change. `Webpacker::Configuration` should now be `Webpacker.config`. Due to this, following the [documentation in the README](https://github.com/reactjs/react-rails/#use-with-webpacker) no longer works.

This is while using Webpacker master (new version hasn't been released yet to include these changes), which is required to [fix an issue](https://github.com/rails/webpacker/issues/673#issuecomment-323592030) with on-demand compiling in development.

**Comment by brentdodell** [comment]
It looks like the generator isn't the only thing that was broken with this. `Webpacker::Manifest` is not `Webpacker.manifest`. It would be good to look through the Webpacker PR linked to above to figure out what all needs to be updated.

**Comment by fresham** [comment]
I noticed this too and was about to make a patch and PR when I saw this issue. I believe it was the release of v3.0.0 of [rails/webpacker](https://github.com/rails/webpacker). [This section](https://github.com/reactjs/react-rails/blob/9ca9a46f64ec601eff08bcdc9bd427348596159c/lib/generators/react/install_generator.rb#L63) is failing for me when running the install generator in a new rails 5.1.3 app with the `--webpack` option.

    Webpacker::Configuration.source_path

should be:

    Webpacker.configuration.source_path

And the lines with `entry_path`:

        .join(Webpacker::Configuration.entry_path)

should use `source_entry_path` instead:

        .join(Webpacker.configuration.source_entry_path

I'd be happy to help with converting to the new API if any maintainers are open to PR's?
